patches/ui-color-color_id.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ui/color/color_id.h b/ui/color/color_id.h
+index c9df38346b84c..7bb58bb01a476 100644
+--- a/ui/color/color_id.h
++++ b/ui/color/color_id.h
+@@ -674,7 +674,7 @@
+ 
+ #define COLOR_IDS \
+   CROSS_PLATFORM_COLOR_IDS \
+-  PLATFORM_SPECIFIC_COLOR_IDS
++  PLATFORM_SPECIFIC_COLOR_IDS MATERIAL_MISSING_COLOR_IDS
```

Comment:
can't you insert `MATERIAL_MISSING_COLOR_IDS` between? like so:
```
   CROSS_PLATFORM_COLOR_IDS \
+  MATERIAL_MISSING_COLOR_IDS \
   PLATFORM_SPECIFIC_COLOR_IDS
```

also it should be named `BRAVE_...`

I believe this way you don't need to "fix" chromium_src checker.

Reply:
Renamed & formatted - Unfortunately I still have to add the entry in `check_chromium_src_config.json5` for it to pass